special-pages/pages/new-tab/messages/types/freemiumPIRBanner-message.json

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Freemium PIR Banner Data",
+    "description": "The 'content' field is optional. Use that fact to show/hide messages",
+    "type": "object",
+    "properties": {
+        "content": {
```

Comment:
can you make `content` required, but then have it’s value be `null` or the object. RMF uses an optional content field but that was just something I missed in the review - next steps and update-notification (and some customizer stuff) use the pattern I’m suggesting. Example here https://github.com/duckduckgo/content-scope-scripts/blob/main/special-pages/pages/new-tab/messages/types/next-steps.json#L9

It affects the generated typescript, and in turn the generated docs. we want the field to be present always, but it’s ok to have it’s value be null.

it’s the difference between 

```ts
content?: SomeType
```

and

```ts
content: SomeType | null
```

Reply:
gotcha. Should we fix RMF or are API contracts already in place for this?